GithubJobApp/Job.swift

```diff
@@ -15,11 +15,11 @@ struct Job {
     let companyUrl: String?
     let description: String
     
-    init(title: String, company: String, description: String, companyLogoUrl: String? = nil, companyUrl: String? = nil) {
-        self.company = company
-        self.title = title
-        self.companyLogoUrl = companyLogoUrl
-        self.description = description
-        self.companyUrl = companyUrl
+    init(jsonJobDictionary: [String: AnyObject]) {
```

Comment:
Another naming note...  you've got a couple of attributes named <something>Url.  I find it easier if names with Url are actually NSURL objects.  What I usually do with decoding JSON like this is to store the string from the server in an attribute called `rawCompanyUrl` (or whatever) and then make the NSURL version of that available through a 'computed property' like this:

```swift
var companyUrl: NSURL? {
     return NSURL(string: rawCompanyUrl)
}
```

Then you can use `job.companyUrl` else where in the code as an NSURL? object.